dev/com.ibm.websphere.appserver.features/test/src/com/ibm/ws/feature/tests/VisibilityTest.java

```diff
@@ -471,6 +471,10 @@ public void testNonTransitiveTolerates() {
         for (Entry<String, FeatureInfo> entry : features.entrySet()) {
             String featureName = entry.getKey();
 
+            if (featureName.contains("versionless")) {
```

Comment:
This also probably also a temp workaround until we resolve Jared's "private visibility" POC issue